dessia_common/workflow/blocks.py

```diff
@@ -112,6 +113,18 @@ def dict_to_object(cls, dict_: JsonSerializable, force_generic: bool = False,
     def evaluate(_):
         return []
 
+    def _to_script(self) -> ToScriptElement:
+        input_variables = [f"Variable(memorize={input.memorize}, name='{input.name}')" for input in self.inputs]
+        inputs = '[' + ",".join(input_variables) + ']'
+        script = f"Display(inputs={inputs}, order={self.order}, name='{self.name}')"
```

Comment:
I am unsure we wish to set inputs this way. A None value for inputs would mean that we are trying to add all the displays of an object. A defined value for inputs would mean it is used as a Base Class and we are for example instantiating a Multiplot Block.

This means we might need to split these two behaviors. I will have a look at it when working on Exports and Displays, soon